split_interleave_fastqs.c

```diff
@@ -8,40 +8,67 @@
 #include <string.h>
 #include <unistd.h>
 
+#define MAX_SUFFIX_LEN 24  // _chunkNNNNNN.fastq
```

Comment:
```
#define MAX_SUFFIX_LEN sizeof("_chunkNNNNNN.fastq")
```

This will be resolved at compile time in any optimizing compiler.

Note that it will include a byte for a trailing `\0`, which isn't a problem in our use case.